### ISSUE TITLE
Any advantage to going to 0057 bios?

### ISSUE THREAD

**Issue by DWomack** [hackintosh-efi/intel-nuc10#75]

**Comment by essereio** [comment]
Security. You can safely upgrade to 0057 after CFG has been unlocked while running bios 0055

**Comment by DWomack** [comment]
For the past 4 releases or so, I have been creating a Boot Flash on 32 Gig  Flash drives using your EFI.  I have four flash USBs sitting on my desk.  I create the flash, copy the EFI, and edit config.plist with my uniqueness.  That's what I boot from and normally run from.  The flash is plugged into the back of the NUC10i7FNH.  I run this machine headless with RealVNC Connect normally.  I have a 4K HDMI dummy plugged in.

This time it would not boot all the way through to where VNC could connect to it.

I turned it off and put the previous 0.82 flash in and it would boot and connect.

I turned it off and the 0.83 back in with a monitor connected and it boots as expected.

For some reason it doesn't like the headless operation.  

Any thoughts?
